content/docs/setup/kubernetes/install/helm/index.md

```diff
@@ -193,21 +163,33 @@ to manage the lifecycle of Istio.
     58
     {{< /text >}}
 
-1. Install the `istio` chart:
+1. Install the `istio` chart. For a production environment use the recommended Helm values or customize as needed:
```

Comment:
Please add a link to the section with the recommended values mentioned.

```suggestion
1. Install the `istio` chart. For a production environment use the 
    [recommended Helm values]() or customize as needed:
```

Reply:
As I mentioned, the recommended values are what we ship with the default helm chart in the line below. Please try to get the bookinfo running and you will understand.